pkg/kubernetes/version.go

```diff
@@ -0,0 +1,220 @@
+package kubernetes
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+	AnnotationSpinnakerArtifactVersion = `artifact.spinnaker.io/version`
+	AnnotationSpinnakerMonikerSequence = `moniker.spinnaker.io/sequence`
+	Annotation
+)
+
+type SpinnakerVersion struct {
+	Long  string
+	Short string
+}
+
+// AddSpinnakerVersionAnnotations adds the `artifact.spinnaker.io/version` and `moniker.spinnaker.io/sequence` annotations to the manifest to identify the version number of that resource
```

Comment:
End this comment with a `.`. Separate it onto a new line since it's kinda long.

Reply:
done